ibc/mpt/mpt_test.go

```diff
@@ -1,63 +0,0 @@
-package mpt
```

Comment:
these tests needed a refactor because I changed the proof reconstruction logic. didn't want to spent too much time on fixing these since most of the light client logic is not tested anyway.